welcome/index.html

```diff
@@ -0,0 +1,351 @@
+        </li>
+        <li class="outlook">
+          <h3 class="headline-content"><span>Firmware Directory</span></h3>
+          <img class="outlook-banner" src="img/outlook-firmware.jpg" alt="firmware directory">
+          <p>That CO2 sensor you always wanted to build and hook up to your wireless network? Somebody already built that! By building up a directory of firmwares and an easy way to flash your devices we'll be able to build upon each others work. That's what it's all about isn't it?</p>
+        </li>
+        <li class="outlook">
+          <h3 class="headline-content"><span>Services</span></h3>
+          <img class="outlook-banner" src="img/outlook-services.jpg" alt="services">
+          <p>That cool app that you just pushed to the experimental platform now needs a simple messaging layer? What if you could simply add a service to provide that? Just like on Heroku, EP provides a simple way to include services and use them in your application.</p>
+        </li>
+      </ol>
+    </div><!-- /.overlook -->
+
+    <div class="technologies section-content">
+      <h2 class="headline-content"><span>Technologies</span></h2>
+
+      <div class="supported-technologies">
+        <ul>
+          <li class="nodejs" title="Build IoT apps with node.js">
+            <figure>
+              <img src="img/nodejs.svg">
+            </figure>
+            <figcaption>Node.JS</figcaption>
+          </li>
+          <li class="rails" title="Build IoT apps with Ruby on Rails">
+            <figure>
+              <img src="img/rails.svg">
+            </figure>
+            <figcaption>Ruby (on Rails)</figcaption>
+          </li>
+          <li class="golang" title="Build IoT apps with Golang">
+            <figure>
+              <img src="img/go.svg">
+            </figure>
+            <figcaption>Go</figcaption>
+          </li>
+          <li class="php" title="Build IoT apps with PHP">
+            <figure>
+              <img src="img/php.svg">
+            </figure>
+            <figcaption>PHP</figcaption>
+          </li>
+          <li class="python" title="Build IoT apps with Python">
+            <figure>
+              <img src="img/python.svg">
+            </figure>
+            <figcaption>Python</figcaption>
+          </li>
+          <li class="dockerfile" title="Build IoT apps with Docker">
+            <figure>
+              <img src="img/docker.svg">
+            </figure>
+            <figcaption>Dockerfile</figcaption>
+          </li>
+        </ul>
+      </div><!-- /.supported-technologies -->
+
+      <p>
+        This project wouldn't be possible without the extensive use of other great open source technologies: Docker, Dokku, XYZ, Node.js, Go, abc and so many more. The code is open source under Apache 2.0. Feel free to build, fork and share.
```

Comment:
I think the text should state that you can write apps in Node.js, Python, PHP, ...